schemas/variant_calling.json

```diff
@@ -269,8 +272,7 @@
               },
               "required": [
                 "target_capture_kit",
-                "primary_target_regions",
-                "capture_target_regions"
+                "primary_target_regions"
```

Comment:
what's the reason for this change?

Reply:
We only know what regions/genes they're targeting but not what they used for validation.